### PR TITLE
Explain how to setup gui_in_3d

### PR DESCRIPTION
**Pull request by idbrii**
Make it more obvious to newcomers how this demo works so they can replicate it in their projects.

Change the text to point to how this is working.

Add Editor Descriptions to make it clearer how this is setup.

image

### CODE REVIEW

**Comment by idbrii** [comment]
Not sure if it's worth mentioning this relevant docs page:
https://docs.godotengine.org/en/stable/tutorials/shaders/using_viewport_as_texture.html
Could add to the readme, but it's only kind of related so maybe that's more confusing?

File: viewport/gui_in_3d/gui_panel_3d.tscn [hunk 1]
```diff
@@ -23,6 +23,7 @@ process_mode = 3
 script = ExtResource("1")
 
 [node name="SubViewport" type="SubViewport" parent="."]
+editor_description = "SubViewport allows us to use a ViewportTexture to display the contents of this viewport."
```
Comment (aaronfranke) [inline viewport/gui_in_3d/gui_panel_3d.tscn:26]:
```suggestion
editor_description = "SubViewport contents can be displayed using a ViewportTexture."
```

**Review by aaronfranke — COMMENTED** [review]
Makes sense.

**Review by aaronfranke — APPROVED** [review]

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**